docker/spec/dependabot/docker/update_checker_spec.rb

```diff
@@ -451,7 +451,7 @@
 
       context "followed by numbers and with less components than other version but higher underscore part" do
         let(:version) { "11.0.16_8-jdk" }
-        it { is_expected.to eq("11.0.16.1_1-jdk") }
+        it { is_expected.to eq("11.0.16_8-jdk") }
```

Comment:
This one update is probably expected so this needs more work.

I think we may want to choose to keep precision when we have to choose between several tags that all point to the same SHA (like, say, `php:8`, `php:8.1`, and `php:8.1.12`) , but allow precision changes other than that.

Reply:
I amended the PR as suggested in the previous review comment. Happy with it now!